forward-pull-requests

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
```

Comment:
Looks like a deprecated script, [last used](https://github.com/search?q=repo%3Acms-sw%2Fcmssw+%22Please+wait+for+a+new+IB%22&type=pullrequests&s=created&o=desc) in 2017